.template/variants/web/Gemfile.rb

```diff
@@ -23,10 +23,11 @@
   EOT
 end
 
-insert_into_file 'Gemfile', after: %r{gem 'pronto-reek'.*\n} do
+insert_into_file 'Gemfile', after: %r{gem 'danger'.*\n} do
   <<-EOT
-  gem 'pronto-scss', require: false
-  gem 'pronto-eslint_npm', require: false
+  gem 'danger-slim_lint' # Lint Slim template files.
+  gem 'danger-eslint' # Eslint
+  gem 'scss_lint' # Scss lint
```

Comment:
```suggestion
  gem 'scss_lint' # SCSS lint
```
Minor thing but I think it would be better, what do you think.

Reply:
Seems legit. Will change it.